### ISSUE TITLE
DDS  time-dependent opti

### ISSUE THREAD

**Issue by Alex-Zhu1** [GiilDe/turbo-edit#2]
HI, i am trring to repreduce your supp-C dds  time-dependent opti. when i set weight about T:

`              

                alpha_t = self.scheduler.alphas[t].to(device)

                alpha_bar_t = self.scheduler.alphas_cumprod[t].to(device)

                bt = 1.0 - alpha_t

                ct = torch.sqrt(alpha_bar_t) * torch.sqrt(1-alpha_bar_t)

                weight = (bt / ct).detach()

                grad = eps_pred # * w_DDS #* weight
                `
i found opti processing dont change the result. can u help me

Thanks !!

**Comment by GiilDe** [comment]
Hi, I just added code that implements this experiment. I believe the piece of code you're looking for is this - https://github.com/GiilDe/turbo-edit/blob/13178ceab2157366364fc45c78b9f61cf00dbec1/dds_ef_eq/dds_ef_eq.py#L261 ? 
Let me know if it answers your question

**Comment by Alex-Zhu1** [comment]
nice!~ i got it.
Thank you! wish better work in the future!
